content/blog/2023/2023-q3-update/index.md

```diff
@@ -0,0 +1,131 @@
+
+This became clear in our first in-person team meeting last May.
+At that meeting, we realized that many on the team were spending too much of their time "reacting" to demands from the service.
+We also learned that the scope and complexity of our various workstreams had gotten to a size where our informal team structures of work prioritization were no longer adequate.
+
+## Growing the complexity of our team to match the complexity of our service
+
+So, for the past several months we've been working on a plan to evolve 2i2c's team structure in order to more effectively manage the complexity of our service, and balance long- and short-term thinking.
+
+This began with an organizational audit carried out by [makeadifference.digital](https://www.makeadifference.digital/), a consulting group that focusese on tech-for-good and non-profit products and services.
+They conducted interviews with everybody on the team, and concluded that we have a few key functions missing that were creating or compounding the stresses people felt.
+
+We're looking into ways that we can make more of this work public, but in the meantime here is an overview of some highlights:
+
+### We need a dedicated product functionality
+
+First, we realized that we have a number of new "signals" pushing our service in many different directions.
+Some are external - from communities we work with or from funders. Some are internal - from different team member visions of where work is needed.
+After growing our service and our team, there are now too many perspectives and voices to balance in a purely unstructured and organic way.
+
+So we're creating a **dedicated "product function" within 2i2c**.
+Its goal will be to serve as an integrator across the many stakeholders that are interested in 2i2c's service, and synthesize a collective product strategy and system of prioritization to move the organization forward.
+They'll help 2i2c grow a culture of iteration and decisions that are driven by the needs of our community partners and users.
+
+We're [hiring a Product Lead](https://2i2c.org/jobs/2023/product-lead/) to kick off this effort.
+If you think you'd be interested in this role, please consider applying!
+
+### We need a more structured system of work and execution
+
+Next, we realized that there are many different workstreams we must balance simultaneously.
+There are new communities to be onboarded, infrastructure bugs to be fixed, questions to be answered, and strategic priorities to improve the service.
+Expecting each team member to individually find the right balance between all of these is increasingly unrealistic.
+We need capacity to oversee this system of work and bring some structure to it.
+
+So, we're creating a **Deliver Manager role** to oversee the system of work that our engineering team uses for its planning and execution.
+In addition, this role will serve at an organization-wide capacity as an **interrim Chief of Staff**.
```

Comment:
In addition, this role will serve at an organization-wide capacity as an **interim Chief of Staff**.